File: dev-superior-algoritmo-e-logica-de-programacao/cpp/exercicios/operadora.cpp
#include <iostream>
#include <iomanip>

using namespace std;

int main() {
    int quantMinutos, minutosExcedentes;
    double valorPagamento;

    cout << "Digite a quantidade de minutos: ";
    cin >> quantMinutos;

    valorPagamento  = 50.0;

    if (quantMinutos  > 100) {
        minutosExcedentes = quantMinutos - 100;
        valorPagamento = valorPagamento + minutosExcedentes * 2.0;
    }

    cout << fixed << setprecision(2);
    cout << "Valor a pagar: R$ " << valorPagamento << endl;

	return 0;
}

File: nelio-alves/algoritmo-e-logica-de-programacao/java/src/exercicios/Operadora.java
package exercicios;

import java.util.Locale;
import java.util.Scanner;

public class Operadora {

	public static void main(String[] args) {

		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);

		int quantMinutos, minutosExcedentes;
		double valorPagamento;

	    System.out.print("Digite a quantidade de minutos: ");
		quantMinutos = sc.nextInt();

		valorPagamento = 50.0;

		if (quantMinutos > 100) {
			minutosExcedentes = quantMinutos - 100;
			valorPagamento = valorPagamento + minutosExcedentes * 2.0;
		}
		
	    System.out.printf("Valor a pagar: R$ %.2f\n", valorPagamento);

		sc.close();
	}
}
